js/_api.js

```diff
@@ -20,21 +20,27 @@ export function call(uri, method, payload) {
 		credentials: 'same-origin',
 		method: method,
 		headers: {
-			Accept: 'application/json',
-			'Content-Type': 'application/json'
+			Accept: 'application/json'
 		}
 	};
 
-	if(payload) {
-		var data = new FormData();
-		data.append("json", JSON.stringify(payload));
-		options.body = JSON.stringify(payload);
+	if(method.toLowerCase() === "post") {
+		options.body = new FormData();
```

Comment:
Massive war? I discovered that the current functionality for token checking is using the built in silverstripe `$request->postVar('asdsd');` and has no ability to easily read json body content from the content. As my trusty peer reviewer and friend, would you prefer?